crypto/asn1/a_strnid.c

```diff
@@ -144,14 +147,15 @@ ASN1_STRING_TABLE *ASN1_STRING_TABLE_get(int nid)
 static ASN1_STRING_TABLE *stable_get(int nid)
 {
     ASN1_STRING_TABLE *tmp, *rv;
+
     /* Always need a string table so allocate one if NULL */
     if (stable == NULL) {
         stable = sk_ASN1_STRING_TABLE_new(sk_table_cmp);
         if (stable == NULL)
             return NULL;
     }
     tmp = ASN1_STRING_TABLE_get(nid);
-    if (tmp && tmp->flags & STABLE_FLAGS_MALLOC)
+    if (tmp != NULL && tmp->flags & STABLE_FLAGS_MALLOC)
```

Comment:
I think this would be more readable as:

`if (tmp != NULL && (tmp->flags & STABLE_FLAGS_MALLOC))`

or (better) as:

`if (tmp != NULL && (tmp->flags & STABLE_FLAGS_MALLOC) != 0)`